keps/sig-release/2572-release-cadence/kep.yaml

```diff
@@ -0,0 +1,37 @@
+title: Defining the Kubernetes Release Cadence
+kep-number: 2572
+authors:
+  - "@justaugustus"
+owning-sig: sig-release
+participating-sigs:
+  - sig-architecture
+  - sig-testing
+status: implementable
+creation-date: 2021-01-21
+reviewers:
+  - "@BenTheElder"
+  - "@derekwaynecarr"
+  - "@dims"
+  - "@ehashman"
+  - "@hasheddan"
+  - "@jeremyrickard"
+  - "@johnbelamaric"
+  - "@spiffxp"
+  - "@stevekuznetsov"
+approvers:
+  - "@LappleApple"
+  - "@saschagrunert"
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: alpha
+
+# The most recent milestone for which work toward delivery of this KEP has been
+# done. This can be the current (upcoming) milestone, if it is being actively
+# worked on.
+latest-milestone: "v1.22"
+
+# The milestone at which this feature was, or is targeted to be, at each stage.
+milestone:
+  alpha: "v1.22"
+  beta: "v1.23"
+  stable: "v1.25"
```

Comment:
What do milestones and stages even mean in the context of a process KEP?

Reply:
I think the milestones and stages are not well aligned for process related KEPs. I think the only real prior art here was LTS and in that case per @jberkus:

> Per the LTS KEP, alpha and beta phases are for planning, automating, and communication.  i.e. prep.  The GA phase is when we actually change the schedule.

It is probably worth formalizing this for future process related things.

Reply:
A similar KEP in terms of it being about a process change: https://github.com/kubernetes/enhancements/tree/master/keps/sig-docs/1326-third-party-content-in-docs

Reply:
Thanks for that extra data point @sftim 👍

Reply:
I'd suggest the KEPs can accomodate this process change if we use alpha as a socialization period where all affected parties are advised, etc.. it would function like a feature gate, it's not on yet, but we're letting you know the intention to make it default exists. Beta would be the initial rollout and then gathering feedback after the test period to make the final decision if it is going to be GA (ie the new default cadence)